server/lib/configuration.js

```diff
@@ -30,6 +30,14 @@ const conf = module.exports = convict({
     env: 'ALLOWED_PARENT_ORIGINS',
     format: Array
   },
+  amplitude: {
+    disabled: {
+      default: false,
+      doc: 'disable amplitude events to stderr',
```

Comment:
Can you change this doc for clarity to `Disable amplitude events`

Reply:
sure, that's a nice one.